src/components/ContactTop/index.jsx

```diff
@@ -1,67 +1,87 @@
-import React, { Component } from 'react';
+import React, { useState, useEffect } from 'react';
+import { useTransition, animated } from 'react-spring';
 import { charityAPI } from '../../clients/charity';
 import './style.scss';
 import Loader from './ContentLoader';
-class ContactTop extends Component {
-  state = {
-    soicilIconsList: [],
-    loading: true,
-    error: false
-  };
+const ContactTop = () => {
+  const [dataState, setDataState] = useState({});
+  const [loadingState, setLoadingState] = useState(true);
+  const [errorState, setErrorState] = useState({
+    error: false,
+    errorMessage: ''
+  });
 
-  componentDidMount() {
+  const getData = () => {
     charityAPI('/socialmedias')
```

Comment:
U should add here setLoadingState(true);
the reason for this when getdata() is called it should setthe state of loading ... this will happen if somene press the retry button if error is there